### DISCUSSION TITLE
What am I benefiting from using this command?

### DISCUSSION THREAD

**Discussion by idelice** [ajeetdsouza/zoxide#734] [General]
Using zsh, I can navigate to project without using`cd` like so: `.config/nvim`, and if I were to use this CLI, I would do: `z .config/nvim` so what am I benefiting from this? 

Don't get me wrong, I'm not trying to bash zoxide. I'm just trying to find a personal valid use case to use it.

**Comment by ajeetdsouza** [comment]
zoxide is a _smarter_ cd command - it's designed to figure out where you want to go without you having to type in the full path.

In case the description in the README is confusing, you may find this video helpful: https://www.youtube.com/watch?v=aghxkpyRVDY
